microservices/fe/settings/common.py

```diff
@@ -2,9 +2,13 @@
 from pathlib import Path
 
 ################# IMPUT SHAPE #############################
-WIDTH = 300
-HEIGHT = 300
-FPS = 24.0
+#DELAY_TIME is used to control de fps
+#Example if DELAY_TIME = 200 then fps = 1000/200
+DELAY_TIME = 200 
+HIGH_VALUE = 100000
+WIDTH = HIGH_VALUE
+HEIGHT = HIGH_VALUE
```

Comment:
Do we really want to have this with 100k width/height?

Reply:
No, this is just to set up with the maximum resolution possible, what happen here is that the fe will try to set with those value and if it can't, will set up the maximum resolution